searx/utils.py

```diff
@@ -45,7 +45,7 @@ def searx_useragent():
     """Return the searx User Agent"""
     return 'searx/{searx_version} {suffix}'.format(
            searx_version=VERSION_STRING,
-           suffix=settings['outgoing'].get('useragent_suffix', ''))
+           suffix=settings['outgoing'].get('useragent_suffix', '')).strip()
```

Comment:
a trailing white space raises a HTTP error on HTTP/1 connection (not HTTP/2).
`'searx/{searx_version} {suffix}'` --> `"searx/0.18.0 "`